scripts/release-manager/cli.js

```diff
@@ -98,7 +98,9 @@ const app = {
       PATH_TO_REPO = config.reactPath;
       return config;
     } catch (e) {
+      /* eslint-disable max-len */
       console.error('Attempt to load config file failed. Please run `init` command for initial setup or make sure ~/.react-release-manager.json is valid JSON. Using a default config which may not work properly.');
+      /* eslint-enable max-len */
```

Comment:
nit: Probably better to split string across lines rather than bypass lint rule (eg like [here](https://github.com/facebook/react/blob/3c75593692524678fbb74e8077018cc39b04ffaa/src/renderers/dom/fiber/ReactDOMFiber.js#L324-L328)).